packages/xrpl/test/integration/transactions/trustSet.test.ts

```diff
@@ -1,14 +1,14 @@
 import { assert } from 'chai'
 
-import { TrustSet, percentToQuality, Wallet } from '../../../src'
+import { TrustSet, percentToQuality, Wallet, TrustSetFlags } from '../../../src'
 import serverUrl from '../serverUrl'
 import {
   setupClient,
   teardownClient,
   type XrplIntegrationTestContext,
 } from '../setup'
 import { generateFundedWallet, testTransaction } from '../utils'
-
+import RippleState from '../../../src/models/ledger/RippleState'
```

Comment:
Shouldn't this be imported from `'../../../src'` ?

Reply:
The path currently in the PR resolves to this relative path: `packages/xrpl/src/models/ledger/RippleState.ts`.

However, `../../../src/index` does not have a default export. It does not recognize the name `RippleState`

Reply:
`ledger` objects are imported under a special alias `LedgerEntry`: https://github.com/XRPLF/xrpl.js/blob/61da4c567aba341b6ec534ff5f6e50a819703a1f/packages/xrpl/src/models/index.ts#L9

This is the reason why I can't import `RippleState` directly from `../../../src`. To use your suggestion, I'll need to use `LedgerEntry.RippleState`.